renderer/pages/index.tsx

```diff
@@ -429,66 +431,94 @@ export default function Home() {
         <div className="grid max-w-3xl grid-cols-1 gap-6 mx-auto sm:px-6 lg:max-w-[1400px] lg:grid-cols-[31%_31%_auto]">
           {reportsFolder ? (
             <>
-              <div className="space-y-6 lg:col-start-1 lg:col-span-2 flex flex-col">
-                <section>
-                  <div className="bg-white shadow sm:rounded-lg dark:bg-dark-container dark:border dark:border-dark-border">
-                    <DateSelector
+              <div className="lg:col-start-1 lg:col-span-2 flex flex-col gap-6">
+                <section className="bg-white shadow sm:rounded-lg dark:bg-dark-container dark:border dark:border-dark-border">
+                  <DateSelector
+                    selectedDate={selectedDate}
+                    setSelectedDate={setSelectedDate}
+                    isDropboxConnected={isDropboxConnected}
+                    selectedDateReport={selectedDateReport}
+                  />
```

Comment:
We have to avoid such huge files and don't allow to do like this. When you start developing some components you have to understand, that the file shouldn't be huge and you can split this file for more components and use it.